mep/people/templates/people/membership_activities.html

```diff
@@ -80,9 +80,9 @@
             <td class="type">{{ event.event_type }}</td>
             <td class="category">{{ event.subscription.category|default:'-' }}</td>
             <td class="duration" data-sort="{{ event.subscription.duration|default:0 }}">{{ event.subscription.readable_duration|default:'-' }}</td>
-            <td class="start" data-sort="{{ event.start_date|date:"Ymd" }}">{{ event.start_date|default:'-' }}</td>
+            <td class="start" data-sort="{{ event.start_date|date:"Ymd" }}">{{ event.partial_start_date|ap_format|default:'-' }}</td>
```

Comment:
We might want a comment here to explain why it's ok to use YMD for sorting but we're using partial dates for display (since that's specific to what we know about the membership data). Or we could just use the partial-date string for sort value....

Reply:
Let's just use the partial string for sorting too, since we are on borrowing activities also. I'll change it.